korman/exporter/mesh.py

```diff
@@ -430,6 +435,10 @@ def _export_geometry(self, bo, mesh, materials, geospans):
                     uvMap[numUVs - 1].normalize()
                     vtx.uvs = uvMap
 
+            # Mark us for blending if we have a alpha vertex paint layer
+            if alpha is not None:
+                geospan.props |= plGeometrySpan.kRequiresBlending
```

Comment:
This seems a little overzealous. What if an artist is in the habit of adding white alpha vcol layers to all their meshes?

Reply:
> What if an artist is in the habit of adding white alpha vcol layers to all their meshes?

You mean like Cyan? 😉

Reply:
> You mean like Cyan? wink

Not the fault of Cyan's artists. This is actually an issue with ZLZ. For various reasons, the PRP format just expects all drawable vertices to have all four RGBA color channels, and ZLZ reflects that when importing Cyan's Ages. So ZLZ will always create two vertex color layers, even when all vertices are fully white or opaque and thus the color/alpha layer is not needed.

I could just tell ZLZ to skip creating those layers when all vertices are white/opaque. This has never bothered me before, but I admit it would make imported Ages much cleaner.